convert2rhel/actions/system_checks/is_loaded_kernel_latest.py

```diff
@@ -54,7 +54,13 @@ def run(self):  # pylint: disable= too-many-return-statements
 
         reposdir = get_hardcoded_repofiles_dir()
         if reposdir and not system_info.has_internet_access:
-            logger.warning("Skipping the check as no internet connection has been detected.")
+            message = "Skipping the check as no internet connection has been detected."
+            logger.warning(message)
+            self.add_message(
+                level="WARNING",
+                id="IS_LOADED_KERNEL_LATEST_CHECK_SKIP",
```

Comment:
Here I would be more specific on why this is being skipped as well 

```suggestion
                id="IS_LOADED_KERNEL_LATEST_NO_INTERNET_ACCESS",
```